### DISCUSSION TITLE
Is this no longer necessary?

### DISCUSSION THREAD

**Discussion by ToRa-51** [jareff-g/ALT-Scann8#142] [Q&A]
Juan, 

On Torulf's site, it is indicated in the file /boot/config.txt to complete the line with ”dtparam=i2c_arm=on” with ”, i2c_arm_baudrate=100000” as on the screenshot below. The question is: is this still necessary, because there are already rules in the UserInterface.py and the Controller.ino to set the baudrate to 400KHz?
<img width="500" alt="Scherm­afbeelding 2024-03-04 om 17 16 31" src="https://github.com/jareff-g/ALT-Scann8/assets/26844127/197a2117-3212-409b-93f4-ff5dee7dbcd4">

Thank you in advance for your reply, 
Ton.

**Comment by jareff-g** [comment]
You need the line to contain only: 'dtparam=i2c_arm=on'.
You should remove the baud rate limitation since ALT-Scann8 makes a heavier use of the I2C bus, so any speed limitation would prevent it from working normally.
Juan
